### ISSUE TITLE
 ERROR: Error on or near line 389

### ISSUE THREAD

**Issue by mmphego** [NicoHood/gpgit#18]
```
GPGit 1.3.3 https://github.com/NicoHood/gpgit

==> 1. Generate a new GPG key
  -> 1.1 Strong, unique, secret passphrase
    See: https://github.com/NicoHood/gpgit#11-strong-unique-secret-passphrase
  -> 1.2 Key generation
    No GPG key registered with Git. Generating a new GPG key.
Enter username: My Name
Enter email: mailme@mail.com

We need to generate a lot of random bytes. It is a good idea to perform
some other action (type on the keyboard, move the mouse, utilize the
disks) during the prime generation; this gives the random number
generator a better chance to gain enough entropy.
We need to generate a lot of random bytes. It is a good idea to perform
some other action (type on the keyboard, move the mouse, utilize the
disks) during the prime generation; this gives the random number
generator a better chance to gain enough entropy.
public and secret key created and signed.

gpg: checking the trustdb
gpg: marginals needed: 3  completes needed: 1  trust model: pgp
gpg: depth: 0  valid:   1  signed:   0  trust: 0-, 0q, 0n, 0m, 0f, 1u
gpg: next trustdb check due at 2020-02-06
    Your new GPG fingerprint is: '9F636BE3AA99E2CCDA4FD1B0DC798C11099B7063'
==> 2. Publish your key
  -> 2.1 Send GPG key to a key server
    Registering new GPG key with Git and uploading it to keyserver 'hkps://pgp.mit.edu'.
==> ERROR: Error on or near line 389. Please report this issue: https://github.com/NicoHood/gpgit/issues
```

**Comment by NicoHood** [comment]
Hi,
could you please try again? Maybe the keyserver was offline? You must upload the key manually now:
https://github.com/NicoHood/gpgit#21-send-gpg-key-to-a-key-server

**Comment by mmphego** [comment]
Hi,

It was a keyserver related issue, works now.
Thanks.
M

**Comment by NicoHood** [comment]
I will leave it open, as I should better print a more detailed error message.

**Comment by NicoHood** [comment]
Fixed in 1.3.4
